packages/@aws-cdk/aws-apigateway/package.json

```diff
@@ -132,6 +135,11 @@
       "lib/apigatewayv2.js"
     ]
   },
+  "pkglint": {
+    "exclude": [
+      "no-experimental-dependencies"
```

Comment:
I need to depend on the apigatewayv2 L1s. This doesn't cause `aws-apigateway` to depend on `aws-apigatewayv2-alpha`, it's just makes you fail faster if you add a dependency to an experimental V2. ubergen will strip out the L2s from `aws-apigatewayv2`.